ci/driver.sh

```diff
@@ -23,48 +23,62 @@ repo_url=${repo_url:-"https://github.com/NOAA-EMC/global-workflow.git"}
 ################################################################
 # Setup the reletive paths to scripts and PS4 for better logging 
 ################################################################
-set -eux
-pwd="$(cd "$(dirname  "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd )"
+script_pwd="$(cd "$(dirname  "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd )"
```

Comment:
```suggestion
ROOT_DIR="$(cd "$(dirname  "${BASH_SOURCE[0]}")/.." >/dev/null 2>&1 && pwd )"
```
Instead of `script_pwd` and then using relative paths, IMO, it is better to find the `ROOT_DIR` and then use that wherever needed.  So you don't have to guess where you are in the script.